php/Terminus/Collections/UserMemberships.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Terminus\Collections;
+use Terminus\Request;
+use Terminus\Session;
+use Terminus\UserMembership;
+use \Terminus_Command;
+
+class UserMemberships {
+  private $site;
+  private $models = array();
+  private $workflows;
+
+  /**
+   * Object constructor. Saves site object
+   *
+   * @return [UserMemberships] $this
+   */
+  public function __construct($options = array()) {
+    $this->site = $options['site'];
+    $this->workflows = new Workflows(array('owner' => $this->site, 'owner_type' => 'site'));
```

Comment:
Same comment as on the model about using  $this->site->workflows collection, rather than attaching workflows to this collection.